### PR TITLE
Added opera mini fix for label font

### LINKED ISSUE #482: Labels small do not work on opera mini
screenshot_20151028-145626

### LINKED ISSUE #489: Labels alignment broken in opera mini
screenshot_20151030-140710

### PR DESCRIPTION
**Pull request by mr-mig**
Fixed #482 
Fixed #489 

Labels::
image

Buttons with labels:
image

### CODE REVIEW

**Comment by kdzwinel** [comment]
ok

File: src/sass/_fixes.scss [hunk 1]
```diff
@@ -0,0 +1,17 @@
+@mixin fixText($fontSize, $lineHeightOffset: 0) {
+  // $lineHeightOffset is a way to fix font issue with the baseline
+  // the issue is visible when vertically centering the text of at least 12px
+  font-size: $fontSize;
+  height: $fontSize;
+  line-height: $fontSize + $lineHeightOffset;
+}
+
+@mixin fixOperaMiniLabelText {
```
Comment (aju) [inline src/sass/_fixes.scss]: maybe we should use Opera Mini specific selector like:
`:-o-prefocus, .block1, .block2 {font-size: 14px;}` ?

Comment (mr-mig) [inline src/sass/_fixes.scss]: Hacky. As soon as it works everywhere, its ok to not wrap it IMO

Comment (aju) [inline src/sass/_fixes.scss]: yeah true

**Comment by aju** [comment]
Lets go with this!!

**Pull request merged**